src/main/kotlin/com/dsrm/dsrmbackend/properties/JwtProperties.kt

```diff
@@ -0,0 +1,14 @@
+package com.dsrm.dsrmbackend.properties
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import java.security.Key
+
+
+@ConfigurationProperties(prefix = "dsrm.auth.jwt")
+class JwtProperties {
+    var expirationMs = 0
```

Comment:
it should be private. And I would prefer if you could use data class instead of class like here in section 3.2
https://www.baeldung.com/kotlin/spring-boot-configurationproperties